lib/membrane_kino.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Membrane.Kino do
```

Comment:
Ideally it would be `KinoMembrane` as per [this guideline](https://hexdocs.pm/kino/Kino.html#module-custom-kinos) :)